### ISSUE TITLE
Error: The template.toml is missing

### ISSUE THREAD

**Issue by Lizhen0797** [sacha-l/kickstart-substrate#8]
when i use the command
```kickstart https://github.com/sacha-l/kickstart-substrate```

An error occurred 
``` Tmp dir: "C:\\Users\\Admin\\AppData\\Local\\Temp\\"
Error: The template.toml is missing ```

**Comment by sacha-l** [comment]
Which version? Make sure you're using kickstart v3. It should work (I used it yesterday in some tests).

**Comment by Lizhen0797** [comment]
> Which version? Make sure you're using kickstart v3. It should work (I used it yesterday in some tests).

I'm using the docs v3 and the kickstart was pulled at 11.22
This problem is caused by mirror. When I changed the mirror, the problem was solved, but a new problem appeared.

i'm trying to learn the tutorial "Build the Substrate Kitties Chain", when I learn the part **_"Write the pallet_kitties scaffold"_** and run this command
``` cargo build -p pallet-kitties ```

the project can't build, the errors are

``` error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
  --> /Users/lizhen/.cargo/git/checkouts/substrate-7e08433d4c370a21/26d69bc/primitives/runtime-interface/src/impls.rs:45:1
   |
45 | assert_eq_size!(usize, u32);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: source type: `usize` (64 bits)
   = note: target type: `u32` (32 bits)
   = note: this error originates in the macro `assert_eq_size` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
  --> /Users/lizhen/.cargo/git/checkouts/substrate-7e08433d4c370a21/26d69bc/primitives/runtime-interface/src/impls.rs:47:1
   |
47 | assert_eq_size!(*const u8, u32);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: source type: `*const u8` (64 bits)
   = note: target type: `u32` (32 bits)
   = note: this error originates in the macro `assert_eq_size` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by sacha-l** [comment]
What template version are you using? Everything's been updated to `monthly-2021-11-1` - if you global search that and find it, you're good. Otherwise you need to get the latest node template (via kickstart or from the [node template repo](https://github.com/substrate-developer-hub/substrate-node-template). 

Another issue I often see is outdated toolchains. Run `rustup show` and if your version is less than `rustc 1.56.1`, please update it.

**Comment by Lizhen0797** [comment]
> 
yeah, the template version is 3 (in the cargo. lock), and every package in pallets/kitties is ```monthly-2021-11-1 ```, Using the ``` rustup show```, the rustic's version in ``` rustc 1.56.1```

![image](https://user-images.githubusercontent.com/58519607/143434819-69b04f26-f21c-4ebb-9f65-4b1bc6172a6a.png)
<img width="1397" alt="截屏2021-11-25 下午7 37 56" src="https://user-images.githubusercontent.com/58519607/143434928-746416f3-23c7-46e3-8d51-1d76bc33582e.png">

**Comment by sacha-l** [comment]
Hm.. you may be missing some instructions? Have you tried with nightly ? 
Do: 
```bash
rustup default stable
rustup update
rustup update nightly
rustup target add wasm32-unknown-unknown --toolchain nightly
```
Then:
`cargo +nightly build -p pallet-kitties` .

**Comment by Lizhen0797** [comment]
I try the way that you said, but it didn't works

**Comment by 99Kies** [comment]
I have met the same problem, I would like to ask whether you have solved the problem?
![1638522198(1)](https://user-images.githubusercontent.com/45918704/144575277-7f07648c-8633-4855-ad23-2dcfa75dabfa.png)

**Comment by Lizhen0797** [comment]
> I have met the same problem, I would like to ask whether you have solved the problem? ![1638522198(1)](https://user-images.githubusercontent.com/45918704/144575277-7f07648c-8633-4855-ad23-2dcfa75dabfa.png)

using the original mirror link, and try many times QAQ

**Comment by sacha-l** [comment]
Try these steps:
1. Clone https://github.com/sacha-l/kickstart-substrate.git.
`git clone https://github.com/sacha-l/kickstart-substrate.git`
2. Go the the template directory.
`cd kickstart-substrate`
3. Run the kickstart command on that directory.
`kickstart .`

**Comment by 99Kies** [comment]
> Try these steps:
> 
> 1. Clone https://github.com/sacha-l/kickstart-substrate.git.
>    `git clone https://github.com/sacha-l/kickstart-substrate.git`
> 2. Go the the template directory.
>    `cd kickstart-substrate`
> 3. Run the kickstart command on that directory.
>    `kickstart .`

So he created a template folder for me, that's what it is, right?  
![image](https://user-images.githubusercontent.com/45918704/144628134-e2039d88-54bf-4a2d-8ca2-fe8a256253b4.png)
